test/index.test.js

```diff
@@ -373,6 +372,19 @@ describe('index', function () {
             });
         });
 
+        it('uses cluster build timeout if user specified a higher timeout', () => {
+            executor.buildTimeout = 20;
+            fakeStartConfig.annotations['beta.screwdriver.cd/timeout'] = 120;
+            postConfig.json.command = [
+                '/opt/sd/launch http://api:8080 http://store:8080 abcdefg 20 15'
```

Comment:
Non-blocking, but it would be preferable to replace the hard-coded `20` with `${executor.buildTimeout}`, similar to the other tests (eg. `${testTimeout}` instead of `45` on line 365).

```js
postConfig.json.command = [
 '/opt/sd/launch http://api:8080 http://store:8080 abcdefg ${executor.buildTimeout} 15'
];
```

Reply:
Would have to split into 2 lines because the line will be too long. That's why I changed it to number... :(
but okay I'll set it to variable.